File: src/main/java/com/learn/guess/GameFor.java
package com.learn.guess;

import org.w3c.dom.ls.LSOutput;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int guess;
        int secret = new Random().nextInt(10)+1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        for (int i = 1;i<=4;i++){
            System.out.print("Please enter number("+i+"/4)：");
            guess = scanner.nextInt();
            if (guess > secret){
                System.out.println("lower");
            }else if (guess < secret){
                System.out.println("higher");
            }else {
                System.out.println("The number is "+secret);
                break;
            }
        }
        System.out.println("結束");
    }
}

File: src/main/kotlin/com/learn/GameFor.kt
package com.learn

import java.util.*
import kotlin.random.Random

fun main() {
    val secret = Random.nextInt(10)+1
    var scanner = Scanner(System.`in`)
    var guess = 0
    println(secret)
    for (i in 1..4){
        print("Please enter number(${i}/4)：")
        guess = scanner.nextInt()
        if (guess>secret){
            println("lower")
        }else if (guess < secret){
            println("higher")
        }else{
            println("The number is $secret")
            break
        }
    }
    println("結束")

    /*for (i in 5 downTo 2){
        print(i)
    }*/
}